webdriver/tests/bidi/storage/set_cookie/page_protocols.py

```diff
@@ -11,8 +11,8 @@
         "https",
     ]
 )
-async def test_page_protocols(bidi_session, get_test_page, domain_value, protocol):
-    set_cookie_result = await bidi_session.storage.set_cookie(cookie=create_cookie(domain=domain_value()))
+async def test_page_protocols(bidi_session, set_cookie, get_test_page, domain_value, protocol):
```

Comment:
One more thing, that I've noticed, somehow during one of the updates we lost the usage of `protocol` value in the test :)

Reply:
let's keep out of the scope of this PR

Reply:
actually, it's a tiny fix. So I added it to this PR